modules/new_profile/php/new_profile.class.inc

```diff
@@ -90,7 +90,12 @@ class New_Profile extends \NDB_Form
         $candidate =& \Candidate::singleton($candID);
 
         if ($config->getSetting('useProjects') == "true") {
-            $candidate->setData('ProjectID', $values['ProjectID']);
+            $candidate->setData(
+                array(
+                 'ProjectID',
+                 $values['ProjectID'],
+                )
```

Comment:
Why is this one not an associative array?

Reply:
You're right, it should be.